daemon/src/projection.rs

```diff
@@ -1238,6 +1233,24 @@ impl From<xtra_bitmex_price_feed::Quote> for Quote {
     }
 }
 
+pub type LatestQuotes = HashMap<ContractSymbol, Quote>;
+
+/// Converts between ContractSymbol types
+fn as_contract_symbol(symbol: &xtra_bitmex_price_feed::ContractSymbol) -> ContractSymbol {
```

Comment:
Why not using `From<xtra_bitmex_price_feed::ContractSymbol> for ContractSymbol` ?

Reply:
I wanted to avoid yet another `ContractSymbol` type (we have 4 already!). Currently one comes from `model` and one from `xtra_bitmex_price_feed`, and both crates don't know about each other, making it impossible to place `From` trait.